deps/rabbitmq_amqp1_0/src/rabbit_amqp1_0_reader.erl

```diff
@@ -240,9 +242,11 @@ close_connection(State = #v1{connection = #v1_connection{
     State#v1{connection_state = closed}.
 
 handle_dependent_exit(ChPid, Reason, State) ->
+    credit_flow:peer_down(ChPid),
+
     case {ChPid, termination_kind(Reason)} of
-        {undefined, uncontrolled} ->
-            exit({abnormal_dependent_exit, ChPid, Reason});
+        %{undefined, uncontrolled} ->
```

Comment:
tidy up.

Reply:
Yeap. this is something that Dializer spotted. It claimed that this case will never happened. I commented it out but yes, I will fully remove it.